src/main/edit_delete/postProcess.ts

```diff
@@ -22,21 +23,23 @@ const removeDeleted = (wordList: Array<any>): void => {
  */
 const FILENAME = 'placeholder_filename.mp4';
 
-const postProcess = (wordList: Array<any>): Array<Clip> => {
```

Comment:
What exactly are we passing here? Is it just the the array of from a Transcription? e.g `Word[]`